narwhal/primary/src/proposer.rs

```diff
@@ -261,12 +263,13 @@ impl Proposer {
                 .observe(batch_inclusion_duration);
         }
 
-        #[cfg(feature = "benchmark")]
         // NOTE: This log entry is used to compute performance.
-        tracing::info!(
-            "Header {:?} was created in {} seconds",
+        debug!(
+            "Header {:?} was created in {} seconds. Contains {} batches, with average delay {} seconds.",
             header.digest(),
-            Duration::from_millis(header.created_at - digests.first().unwrap().2).as_secs_f64()
+            Duration::from_millis(header.created_at - digests.first().unwrap().2).as_secs_f64(),
```

Comment:
Looks like we can no longer just do an unwrap now that it is outside the benchmark. Need to handle the case where there are empty headers

Reply:
Good point! Thought digests cannot be empty but they can.